DragaliaAPI/Features/Quest/QuestController.cs

```diff
@@ -107,6 +126,71 @@ QuestGetQuestClearPartyMultiRequest request
         );
     }
 
+    [HttpPost("open_treasure")]
```

Comment:
Please can you add some automated tests of this endpoint to validate your changes?